features/messages/impl/src/main/res/values/localazy.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+  <string name="screen_room_attachment_source_camera">"Camera"</string>
+  <string name="screen_room_attachment_source_camera_photo">"Take photo"</string>
+  <string name="screen_room_attachment_source_camera_video">"Record a video"</string>
+  <string name="screen_room_attachment_source_files">"Attachment"</string>
+  <string name="screen_room_attachment_source_gallery">"Photo &amp; Video Library"</string>
+  <string name="screen_room_error_failed_processing_media">"Failed processing media to upload, please try again."</string>
+</resources>
```

Comment:
I would not use `screen_room_` prefix for those string, maybe something more generic like `screen_attachment_picker_`. And move to a new module, see my other remark.

Reply:
I wanted to use some `common_*` strings for these, but we agreed with the iOS team to use `screen_room_` and then duplicate & link the strings if needed in Localazy.